lib/main/atomts.ts

```diff
@@ -285,35 +282,7 @@ function readyToActivate() {
 }
 
 export function activate(state: PackageState) {
-
-    // Don't activate if we have a dependency that isn't available
-    var linter = apd.require('linter');
-    var acp = apd.require('autocomplete-plus');
-
-    if (!linter || !acp) {
-        var notification = atom.notifications.addInfo('AtomTS: Some dependencies not found. Running "apm install" on these for you. Please wait for a success confirmation!', { dismissable: true });
-        apd.install(function() {
-            atom.notifications.addSuccess("AtomTS: Dependencies installed correctly. Enjoy TypeScript \u2665", { dismissable: true });
```

Comment:
Would there be a way to get this notification back? As its there in our readme :)